### ISSUE TITLE
Unexpected inputs: sh_checker_comment/sh_checker_shfmt_disable

### ISSUE THREAD

**Issue by kenorb** [luizm/action-sh-checker#17]
As per README, I've added the following inputs:

```
      - name: Shell Check
        uses: luizm/action-sh-checker@master
        with:
          sh_checker_comment: true
          sh_checker_shfmt_disable: true
```

However they're not defined in "action.yml" and GitHub complains about it in _Annotations_ section:

> Unexpected input 'sh_checker_comment', valid inputs are ['']

> Unexpected input 'sh_checker_shfmt_disable', valid inputs are ['']

Build: [113278948](https://github.com/EA31337/EA-Tester/actions/runs/113278948)

**Comment by luizm** [comment]
Thank you, I've never seen this warning before. 

I will fix it and release it soon.
